builtin/types/transfer_gateway/transfer_gateway.proto

```diff
@@ -535,4 +534,4 @@ message TransferGatewayResubmitWithdrawalError {
 
 message TransferGatewayUpdateBinanceTransferFeeRequest{
     BigUInt transfer_fee = 1;
-}
\ No newline at end of file
+}
```

Comment:
Seeing as the protobufs didn't need changing after all revert any changes to this file with `git checkout master -- builtin/types/transfer_gateway.proto` & same with the `.pb.go`.